### PR TITLE
Setup pom, configure .gitignore and create application

### PR DESCRIPTION
**Pull request by grevend-bosch**
- [x] Create and configure the Maven pom.xml file
- [x] Create .gitignore file
- [x] Add sbom generation template
- [x] Add application entry point (main class)

**Disclaimer: This issue indicated that our current code formatting hook is not compatible with JDK16 + class files. The hook has therefore been removed in addition to the points already mentioned.**

### CODE REVIEW

**Comment by grevend-bosch** [comment]
@sbckr blocked by #15

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -5,11 +5,6 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 repos:
-- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
```
Comment (sbckr) [inline .pre-commit-config.yaml:8]: @grevend-bosch 
Please provide more prominent notice of such changes that are in conflict with our automatic code formatting defaults.
The reason for this change is not transparent to everyone, as the rationale is hidden in one of 64 commit titles :wink:

Please create a new ticket to re-enable automated code formatting with git pre-commit hooks and link to the [language-formatter issue](https://github.com/macisamuele/language-formatters-pre-commit-hooks/issues/72).

Thank you

**Review by sbckr — APPROVED** [review]
:+1:
Nevertheless, please have a look at the comment on [.pre-commit-config.yaml](.pre-commit-config.yaml).

File: .pre-commit-config.yaml (hunk 1 above)
Comment (grevend-bosch) [reply .pre-commit-config.yaml:8]: I've added a disclaimer and, as mentioned, I will open a new issue to track the progress.

Comment (grevend-bosch) [reply .pre-commit-config.yaml:8]: See issue #18

**Pull request merged**